app/src/main/res/values/strings.xml

```diff
@@ -14,8 +13,18 @@
     <string name="genres_rock">Rock</string>
     <string name="genres_edm">EDM</string>
     <string name="genres_electronic">Electronic</string>
-
     <string name="image_track">image_track</string>
     <string name="image_more_info">moreinfo</string>
+    <string name="add_album">Add Album</string>
+    <string name="name_album">Name Album</string>
+    <string name="err_name_exist">Album name  is exist !!</string>
+    <string name="err_name_null">Name album is empty</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.err_name_null` appears to be unused
 Unused resources make applications larger and slow down builds.